samples/net/udp/sample.yaml

```diff
@@ -0,0 +1,19 @@
+sample:
+  name: UDP sample
+tests:
+  sample.net.udp:
+    build_only: true
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
+      - thingy91_nrf9160_ns
+      - nrf7002dk_nrf5340_cpuapp
+    platform_allow:
+      - nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
+      - thingy91_nrf9160_ns
+      - nrf7002dk_nrf5340_cpuapp
```

Comment:
Shall we build for qemu as well since we provide a config for it?

Reply:
We can do that, don't see why not.